lib/tooltip_widget.dart

```diff
@@ -35,12 +36,19 @@ class ToolTipWidget extends StatelessWidget {
     this.contentWidth,
   });
 
+  @override
+  _ToolTipWidgetState createState() => _ToolTipWidgetState();
+}
+
+class _ToolTipWidgetState extends State<ToolTipWidget> {
+  Offset position;
+
   bool isCloseToTopOrBottom(Offset position) {
     double height = 120;
-    if (contentHeight != null) {
-      height = contentHeight;
+    if (widget.contentHeight != null) {
+      height = widget.contentHeight;
     }
```

Comment:
This could be combined with the use of `??` operator for more readability.

Reply:
Added ?? operator